x-pack/plugins/monitoring/server/kibana_monitoring/bulk_uploader.js

```diff
@@ -181,7 +179,7 @@ export class BulkUploader {
     const flat = Object.keys(typesNested).reduce((accum, type) => {
       return [
         ...accum,
-        { index: { _type: type } },
+        { index: {} },
```

Comment:
This will actually break internal collection. It's a bit confusing because the endpoint we use to index monitoring documents is a different endpoint than the typical bulk endpoint. This `_type` is actually rewritten in this custom ES endpoint to not be the `type` of the document, but rather a `type` field nested in the `source` of the indexed document. We need to remove this change.

Reply:
@chrisronline in that case what is a way to avoid a deprecation warning?

Reply:
@chrisronline  @sulemanof  
I've created issue to track that:  #32458.